.github/workflows/automerge.yml

```diff
@@ -10,34 +10,63 @@ jobs:
     name: Merge Master to Staging
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set Git config
         run: |
           git config --local user.email "Sofie-Automation@users.noreply.github.com"
           git config --local user.name "GitHub Actions # Sofie-Automation"
       - name: Merge 'master' to 'staging'
+        id: merge
+        continue-on-error: true
         run: |
           git fetch --unshallow
           git checkout staging
           git pull
           git merge --no-ff origin/master -m "Automerge 'master' to 'staging'"
           git push
+      - name: Abort merge
+        if: steps.merge.outcome == 'failure'
+        run: git merge --abort
+      - name: Create Pull Request on merge failure
+        if: steps.merge.outcome == 'failure'
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: master
+          destination_branch: staging
+          pr_title: Automerge Failed
```

Comment:
Would be nice if the title also included the from and to branch. So for exampe: "Automerge Master to Staging failed".
Then when this shows up in the PR overview we know immediatly what it is about :)